### PR TITLE
Resource kind not passed to the loaded component

### PR DESCRIPTION
**Pull request by jhadvig**
Was hitting issue upon editing resources with edit forms, eg. Secrets or RoleBindings,...
@alecmerdler I saw that you have been working on https://github.com/openshift/console/pull/358 which intoduced the `LazyRoute`
PTAL

### CODE REVIEW

**Comment by spadgett** [comment]
jenkins rebuild

File: frontend/integration-tests/views/crud.view.ts [hunk 1]
```diff
@@ -28,6 +32,31 @@ export const filterForName = async(name: string) => {
   await textFilter.sendKeys(name);
 };
 
+export const EditHumanizedKind = (kind) => {
```
Comment (alecmerdler) [inline frontend/integration-tests/views/crud.view.ts]: `camelCase`

File: frontend/integration-tests/tests/crud.scenario.ts [hunk 2]
```diff
@@ -125,6 +125,13 @@ describe('Kubernetes resource CRUD operations', () => {
         expect(crudView.resourceTitle.getText()).toEqual(testName);
       });
 
+      it('edit the resource instance', async() => {
+        await browser.get(`${appHost}/search/${namespaced ? `ns/${testName}` : 'all-namespaces'}?kind=${kind}&q=${testLabel}%3d${testName}`);
+        await crudView.filterForName(testName);
+        await crudView.resourceRowsPresent();
+        await crudView.editRow(kind)(testName);
```
Comment (alecmerdler) [inline frontend/integration-tests/tests/crud.scenario.ts]: Tests are failing because not every resource has an `Edit` option.

**Comment by spadgett** [comment]
jenkins rebuild

**Comment by jhadvig** [comment]
@alecmerdler I've updated the e2e tests... excluded the `ServiceAccount` from the `edit the resource instance` since it can't be edited.
PTAL

**Comment by spadgett** [comment]
Is there a reason the service account isn't editable?

**Pull request merged**